packages/graphql-server/src/functions/graphql.ts

```diff
@@ -276,7 +299,7 @@ export const createGraphQLHandler = ({
     // Only allow execution of specific operation types
     useFilterAllowedOperations(allowedOperations || ['mutation', 'query']),
     // Prevent unexpected error messages from leaking to the GraphQL clients.
-    useMaskedErrors(),
+    // useMaskedErrors(),
```

Comment:
Note not using masked error at moment since this is not compatible when we use the Apollo Server error compatibility plugin.

It masks all non EnvelopErrors and as such masks all errors now.

Guild is working to make them play nicely and will update when there is a new plugin

Reply:
I have them working together now -- the sequence of the plugin declaration is super important. Had to log first, then convert to apollo error, then mask.